index.js

```diff
@@ -33,7 +34,8 @@ exports.register = (server, options) => {
   server.ext({
     type: 'onRequest',
     method(request, h) {
-      request.sentryScope = new Sentry.Scope();
+      Domain.create();
```

Comment:
Domain is not defined. I think this should be `domain`

see the test logs: https://travis-ci.com/hydra-newmedia/hapi-sentry/jobs/205971090

Reply:
Yep, it was too illustrative.  Now it should be also working..  will do some testing now.

Reply:
Did one more update, create() was not enough to active the domain. Now it creates the domain and its giving me breadcrumps only for that request, which is good.

Reply:
The test still fail though. You can just run `npm test` locally. you don't have to push. I don't mind if this code is not up to date and broken.

Reply:
@fiws yep, was trying to figure out how it should be done and wanted to share it early in case you have some insights/feedback. Tests are now passing.